java/src/main/java/com/example/App.java

```diff
@@ -0,0 +1,98 @@
+package com.example;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Map;
+
+import com.sigopt.Sigopt;
+import com.sigopt.exception.APIException;
+import com.sigopt.model.Bounds;
+import com.sigopt.model.Experiment;
+import com.sigopt.model.Observation;
+import com.sigopt.model.Parameter;
+import com.sigopt.model.SuggestData;
+import com.sigopt.model.Suggestion;
+
+class Result
+{
+    Double value;
+    Double stddev;
+
+    public Result(Double value, Double stddev)
+    {
+        this.value = value;
+        this.stddev = stddev;
+    }
+}
+
+public class App
+{
+    public static void main(String[] args) throws APIException, InterruptedException
+    {
+        String clientId = null;
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equals("--client_token") && i < args.length - 1) {
```

Comment:
There are some third party ones, but for just these few arguments this was easy enough